tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -43,9 +43,17 @@
   },
```

Comment:
<ul><li>... labeled <code>dmm_id</code> in the device <em>internally</em> configures a detuning map ...</li><li>... distributing the applied detuning equally over all the masked qubits.</li><li>For instance in the last example qubits "q<strong>1</strong>" and "q2"...</li><li>...&nbsp;and to the trap locations of "q<strong>1</strong>" and "q2"...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>